### ISSUE TITLE
20191001 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#166]
# 20191001 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [qianguyihao/Web](https://github.com/qianguyihao/Web) （Web前端入门和进阶学习笔记，超详细的前端学习图文教程。从零开始学前端，做一个Web全栈工程师。持续更新...）
* [iptv-org/iptv](https://github.com/iptv-org/iptv) （Collection of 8000+ publicly available IPTV channels from all over the world）
* [azl397985856/leetcode](https://github.com/azl397985856/leetcode) （leetcode题解，记录自己的leecode解题之路。）
* [facebook/react](https://github.com/facebook/react) （A declarative, efficient, and flexible JavaScript library for building user interfaces.）
* [sveltejs/svelte](https://github.com/sveltejs/svelte) （The magical disappearing UI framework）
* [b3log/baidu-netdisk-downloaderx](https://github.com/b3log/baidu-netdisk-downloaderx) （⚡️ 一款图形界面的百度网盘不限速下载器，支持 Windows、Linux 和 Mac。）
* [algorithm-visualizer/algorithm-visualizer](https://github.com/algorithm-visualizer/algorithm-visualizer) （&#x1f386;Interactive Online Platform that Visualizes Algorithms from Code）
* [airbnb/javascript](https://github.com/airbnb/javascript) （        JavaScript Style Guide      ）
* [nuxt/nuxt.js](https://github.com/nuxt/nuxt.js) （        Framework for server-rendered Vue.js applications.      ）
* [yaronn/blessed-contrib](https://github.com/yaronn/blessed-contrib) （Build terminal dashboards using ascii/ansi art and javascript）
* [Advanced-Frontend/Daily-Interview-Question](https://github.com/Advanced-Frontend/Daily-Interview-Question) （工作日每天一道前端大厂面试题，祝大家天天进步，一年后会看到不一样的自己。）
* [openlayers/openlayers](https://github.com/openlayers/openlayers) （OpenLayers）
* [haltu/muuri](https://github.com/haltu/muuri) （        Responsive, sortable, filterable and draggable grid layouts      ）
* [DIYgod/RSSHub](https://github.com/DIYgod/RSSHub) （? 使用 RSS 连接全世界）
* [jamiebuilds/the-super-tiny-compiler](https://github.com/jamiebuilds/the-super-tiny-compiler) （⛄️ Possibly the smallest compiler ever）
* [davidshimjs/qrcodejs](https://github.com/davidshimjs/qrcodejs) （Cross-browser QRCode generator for javascript）
* [d2-projects/d2-admin](https://github.com/d2-projects/d2-admin) （An elegant dashboard built by vue）
* [MichMich/MagicMirror](https://github.com/MichMich/MagicMirror) （MagicMirror² is an open source modular smart mirror platform. With a growing list of installable modules, the MagicMirror² allows you to convert your hallway or bathroom mirror into your personal assistant.）
* [jackmoore/autosize](https://github.com/jackmoore/autosize) （Autosize is a small, stand-alone script to automatically adjust textarea height to fit text.）
* [Binaryify/NeteaseCloudMusicApi](https://github.com/Binaryify/NeteaseCloudMusicApi) （        网易云音乐nodejs api      ）
* [duxianwei520/react](https://github.com/duxianwei520/react) （React+webpack+redux+ant design+axios+less全家桶后台管理框架）
* [discordjs/discord.js](https://github.com/discordjs/discord.js) （A powerful JavaScript library for interacting with the Discord API）

## Vue
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [ElemeFE/element](https://github.com/ElemeFE/element) （        A Vue.js 2.0 UI Toolkit for Web      ）
* [bailicangdu/vue2-elm](https://github.com/bailicangdu/vue2-elm) （        基于 vue2 + vuex 构建一个具有 45 个页面的大型单页面应用      ）
* [liuzhuoling2011/baidupcs-web](https://github.com/liuzhuoling2011/baidupcs-web) （基于vue2+nuxt构建的高仿饿了么（2018版））
* [Molunerfinn/PicGo](https://github.com/Molunerfinn/PicGo) （A simple &amp; beautiful tool for pictures uploading built by electron-vue）
* [liyasthomas/postwoman](https://github.com/liyasthomas/postwoman) （&#x1f47d; API request builder - Helps you create your requests faster, saving you precious time on your development.）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [iview/iview-admin](https://github.com/iview/iview-admin) （Vue 2.0 admin management system template based on iView）
* [salomonelli/best-resume-ever](https://github.com/salomonelli/best-resume-ever) （? ? Build fast ? and easy multiple beautiful resumes and create your best CV ever! Made with Vue and LESS.）
* [weilanwl/ColorUI](https://github.com/weilanwl/ColorUI) （鲜亮的高饱和色彩，专注视觉的小程序组件库）
* [ElemeFE/mint-ui](https://github.com/ElemeFE/mint-ui) （        Mobile UI elements for Vue.js      ）
* [iview/iview](https://github.com/iview/iview) （        A high quality UI Toolkit with both Vue.js 2.0 and 1.0      ）
* [sendya/ant-design-pro-vue](https://github.com/sendya/ant-design-pro-vue) （??&#x200d;???&#x200d;? Use Ant Design Vue like a Pro! Demo: https://pro.loacg.com | React: http://pro.ant.design/）
* [bestony/logoly](https://github.com/bestony/logoly) （A Pornhub Pattern Logo Generator）
* [hinesboy/mavonEditor](https://github.com/hinesboy/mavonEditor) （mavonEditor - A markdown editor based on Vue that supports a variety of personalized features）
* [elunez/eladmin-qd](https://github.com/elunez/eladmin-qd) （eladmin前端源码，项目基于 Spring Boot 2.1.0 、 Spring Boot Jpa、 Spring Security、Redis、Vue的前后端分离后台管理系统， 权限控制采用 RBAC，菜单动态路由）

## HTML
* [fengdu78/Coursera-ML-AndrewNg-Notes](https://github.com/fengdu78/Coursera-ML-AndrewNg-Notes) （吴恩达老师的机器学习课程个人笔记）
* [fengdu78/deeplearning_ai_books](https://github.com/fengdu78/deeplearning_ai_books) （deeplearning.ai（吴恩达老师的深度学习课程笔记及资源））
* [ColorlibHQ/AdminLTE](https://github.com/ColorlibHQ/AdminLTE) （AdminLTE - Free Premium Admin control Panel Theme Based On Bootstrap 3.x）
* [crweiner/hacktoberfest-swag-list](https://github.com/crweiner/hacktoberfest-swag-list) （Multiple companies give out swag for Hacktoberfest, and this repo tries to list them all.）
* [PKUanonym/REKCARC-TSC-UHT](https://github.com/PKUanonym/REKCARC-TSC-UHT) （清华大学计算机系课程攻略 Guidance for courses in Department of Computer Science and Technology, Tsinghua University）
* [Igglybuff/awesome-piracy](https://github.com/Igglybuff/awesome-piracy) （A curated list of awesome warez and piracy links）
* [nndl/nndl.github.io](https://github.com/nndl/nndl.github.io) （《神经网络与深度学习》 Neural Network and Deep Learning）
* [AliceWonderland/hacktoberfest](https://github.com/AliceWonderland/hacktoberfest) （Participate in Hacktoberfest by contributing to any Open Source project on GitHub! Here is a starter project for first time contributors. #hacktoberfest）

## CSS
* [daneden/animate.css](https://github.com/daneden/animate.css) （        A cross-browser library of CSS animations. As easy to use as an easy thing.      ）
* [FreeCodeCampChina/freecodecamp.cn](https://github.com/FreeCodeCampChina/freecodecamp.cn) （        看源码请到Code，提问请到Issues，提交代码请到Pull requests，看学习心得请到Wiki。多谢合作，么么哒，^_^。      ）
* [PyConChina/2019-Slides](https://github.com/PyConChina/2019-Slides) （PyCon China 2019 幻灯片）
* [ryanoasis/nerd-fonts](https://github.com/ryanoasis/nerd-fonts) （? Iconic font aggregator, collection, and patcher: 39 patched fonts (87k+ variations). Includes popular glyph collections such as Font Awesome &amp; fonts such as Hack）
* [jgthms/bulma](https://github.com/jgthms/bulma) （        Modern CSS framework based on Flexbox      ）
* [tabler/tabler-react](https://github.com/tabler/tabler-react) （React components and demo for the Tabler UI theme.）
* [Microsoft/TypeScript-Node-Starter](https://github.com/Microsoft/TypeScript-Node-Starter) （        A starter template for TypeScript and Node with a detailed README describing how to use the two together.      ）
* [Tencent/weui](https://github.com/Tencent/weui) （A UI library by WeChat official design team, includes the most useful widgets/modules in mobile web applications.）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [zhaoolee/ChineseBQB](https://github.com/zhaoolee/ChineseBQB) （&#x1f1e8;&#x1f1f3;Chinese sticker pack,More joy / 表情包的博物馆, Github最有毒的仓库, 中国表情包大集合, 聚欢乐~）
* [nikitavoloboev/my-mac-os](https://github.com/nikitavoloboev/my-mac-os) （? A list of applications, alfred workflows and various tools that make my macOS experience even more amazing）
* [Huxpro/huxpro.github.io](https://github.com/Huxpro/huxpro.github.io) （        My Blog / Jekyll Themes / PWA      ）
* [blinkfox/hexo-theme-matery](https://github.com/blinkfox/hexo-theme-matery) （A beautiful hexo blog theme with material design and responsive design.）

## Unknow
* [joshbuchea/HEAD](https://github.com/joshbuchea/HEAD) （        A list of everything that could go in the &lt;head&gt; of your document      ）
* [hakluke/how-to-exit-vim](https://github.com/hakluke/how-to-exit-vim) （Below are some simple methods for exiting vim.）
* [Vay-keen/Machine-learning-learning-notes](https://github.com/Vay-keen/Machine-learning-learning-notes) （周志华《机器学习》又称西瓜书是一本较为全面的书籍，书中详细介绍了机器学习领域不同类型的算法(例如：监督学习、无监督学习、半监督学习、强化学习、集成降维、特征选择等)，记录了本人在学习过程中的理解思路与扩展知识点，希望对新人阅读西瓜书有所帮助！）
* [MunGell/awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) （        A list of awesome beginners-friendly projects.      ）
* [thunlp/PLMpapers](https://github.com/thunlp/PLMpapers) （Must-read Papers on pre-trained language models.）
* [abhat222/Data-Science&#8211;Cheat-Sheet](https://github.com/abhat222/Data-Science--Cheat-Sheet) （Cheat Sheets）
* [getify/You-Dont-Know-JS](https://github.com/getify/You-Dont-Know-JS) （        A book series on JavaScript. @YDKJS on twitter.      ）
* [psibi/rust-book-summary](https://github.com/psibi/rust-book-summary) （Summary of the Rust book）
* [Ignitetechnologies/Privilege-Escalation](https://github.com/Ignitetechnologies/Privilege-Escalation) （This cheasheet is aimed at the CTF Players and Beginners to help them understand the fundamentals of Privilege Escalation with examples.）
* [firstcontributions/first-contributions](https://github.com/firstcontributions/first-contributions) （?✨ Help beginners to contribute to open source projects）
* [kelseyhightower/kubernetes-the-hard-way](https://github.com/kelseyhightower/kubernetes-the-hard-way) （Bootstrap Kubernetes the hard way on Google Cloud Platform. No scripts.）
* [gregsramblings/google-cloud-4-words](https://github.com/gregsramblings/google-cloud-4-words) （The Google Cloud Developer's Cheat Sheet）
* [lydiahallie/javascript-questions](https://github.com/lydiahallie/javascript-questions) （A long list of (advanced) JavaScript questions, and their explanations ✨ Updated weekly!）
* [infosecn1nja/Red-Teaming-Toolkit](https://github.com/infosecn1nja/Red-Teaming-Toolkit) （A collection of open source and commercial tools that aid in red team operations.）
* [HiddenStrawberry/Crawler_Illegal_Cases_In_China](https://github.com/HiddenStrawberry/Crawler_Illegal_Cases_In_China) （Collection of China illegal cases about web crawler 本项目用来整理所有中国大陆涉诉与违规相关的新闻、资料与法律法规。致力于帮助在中国大陆工作的爬虫行业从业者了解我国相关法律，避免触碰数据合规红线。）
* [google/deepdream](https://github.com/google/deepdream) （）
* [dgkanatsios/CKAD-exercises](https://github.com/dgkanatsios/CKAD-exercises) （A set of exercises to prepare for Certified Kubernetes Application Developer exam by Cloud Native Computing Foundation）
* [six2dez/OSCP-Human-Guide](https://github.com/six2dez/OSCP-Human-Guide) （My own OSCP guide）
* [Steve-ShadowsocksR/-SSR-](https://github.com/Steve-ShadowsocksR/-SSR-) （欢迎来到小飞机纯公益SSR分享github免翻墙代理地址,纯公益节点不定期更新!）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [加我微信，进500人微信群，领100本前端电子书](http://t.cn/EV7pcM1)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
